src/app/component-library/policy-editor/editor/policy-form-expression-multi/policy-form-expression-multi.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="flex items-center gap-4 h-[4rem]">
+  <span
+    class="font-bold grow"
+    [matTooltip]="expr.multiExpression!.description"
+    >{{ expr.multiExpression!.title }}</span
+  >
```

Comment:
I see you use `font-bold` here. Right now bc Open Sans does not have any real font weight combinations you can use this and it looks like `font-medium`. When merging the edit/create asset page PR this will become actual Inter font-bold and look too much likely. We just have to remember that